sdk/compute/Azure.ResourceManager.Compute/tests/Scenario/VirtualMachineExtensionVirtualMachineContainerTests.cs

```diff
@@ -0,0 +1,96 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading.Tasks;
+using Azure.Core.TestFramework;
+using Azure.ResourceManager.Compute.Tests.Helpers;
+using NUnit.Framework;
+
+namespace Azure.ResourceManager.Compute.Tests
+{
+    public class VirtualMachineExtensionVirtualMachineContainerTests : VirtualMachineTestBase
+    {
+        public VirtualMachineExtensionVirtualMachineContainerTests(bool async)
+            : base(async)// , RecordedTestMode.Record)
+        {
+        }
+
+        private VirtualMachineExtensionData BasicVirtualMachineExtensionData
+        {
+           get
+           {
+                return ResourceDataHelper.GetBasicLinuxVirtualMachineExtensionData(DefaultLocation);
+           }
+        }
```

Comment:
We could remove this property and replace wherever is invoking this property with the function itself `ResourceDataHelper.GetBasicLinuxVirtualMachineExtensionData(DefaultLocation)`